cli/tests/testdata/lint/expected_glob.out

```diff
@@ -1,3 +1,3 @@
 [WILDCARD]
-Found 3 problems
-Checked 3 files
+Found 6 problems
+Checked 6 files
```

Comment:
This test needs to be fixed, it should change if new test cases are added. I will look into it in a separate PR.

Reply:
Waiting on https://github.com/denoland/deno/pull/12317

Reply:
Resolved now.